python/pyhail/dataset.py

```diff
@@ -1573,15 +1575,14 @@ def split_multi(self, propagate_gq=False, keep_star_alleles=False):
 
         ``split_multi`` adds the following annotations:
```

Comment:
Can you make this a link to the function? 

:py:meth:\`~pyhail.VariantDataset.split_multi\`